setup.py

```diff
@@ -1,11 +1,19 @@
 import os
 import re
 import sys
+from distutils.command import build_ext
 from importlib.machinery import SourceFileLoader
 from pathlib import Path
 
 from setuptools import setup
 
+if os.name == 'nt':
+
+    def pass_ges(self, ext):
+        pass
+
+    build_ext.get_export_symbols = pass_ges
```

Comment:
according to https://bugs.python.org/issue35893 this should be 

```py
build_ext.build_ext.get_export_symbols = ...
```

Also they have a different function for `get_export_symbols`.

Reply:
Good point, I didn't look at the bug. The Stackoverflow answer is what suggested the pass (as the "simplest" option), but I'll try the solution in the bug.